koa-crud/src/types/index.ts

```diff
@@ -56,6 +54,26 @@ type VendorStore = {
   deleteOneVendor: (filters: VendorFilters) => Promise<boolean>;
 };
 
+type Member = {
+  _id?: string;
+  username: string;
+  password: string;
+  realName: string;
```

Comment:
since realName is optional or null. Please change this into:
`realName?: string | null`

Reply:
noted sir